openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -1517,8 +1517,8 @@ def _assert_reg_field(self, extra_fields_setting, expected_field):
         form_desc = json.loads(response.content)
         self.assertIn(expected_field, form_desc["fields"])
 
-
 @ddt.ddt
+@skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Account APIs are only supported in LMS')
```

Comment:
I'm still confused about why this started failing (the email opt in has its own endpoint, and actually is available in CMS). Can you check again to see why this is necessary?